Jenkinsfile

```diff
@@ -105,7 +105,7 @@ pipeline {
         ]) {
           sh "git clone https://github.com/mesosphere/dcos-commons.git ../dcos-commons"
           sh "tar czf oss-build.tar.gz dist"
```

Comment:
🤔

Reply:
not really, but then again there is the other version branch fixing this. either we merge this now or close in favor of real versions.